app/views/finders/_results.mustache

```diff
@@ -22,9 +22,15 @@
 </ul>
 
 {{#zero_results}}
-  <div class='no-results'>
-    <h2 class='gem-c-heading'>There are no matching results.</h2>
-    <p>Try making your search broader.</p>
+  <div class='no-results govuk-!-font-size-19'>
+    <p class='govuk-body govuk-!-font-weight-bold'>There are no matching results.</p>
+    <p class='govuk-body'>Improve your search results by:</p>
+    <ul class="govuk-list">
```

Comment:
You could use the classes `govuk-list govuk-list--bullet` here, and then remove the dashes from the list items.

Reply:
https://design-system.service.gov.uk/styles/typography/

Reply:
the bullets seem to mess up the formatting? they end up sitting outside of the block, in its padding